src/stats.c

```diff
@@ -433,6 +434,16 @@ stats_arena_print(void (*write_cb)(void *, const char *), void *cbopaque,
 		    "resident:                %12zu\n", resident);
 	}
 
+	CTL_M2_GET("stats.arenas.0.tcache_bytes", i, &tcache_bytes, size_t);
+	if (json) {
+		malloc_cprintf(write_cb, cbopaque,
+		    "\t\t\t\t\"tcache\": %zu,\n", tcache_bytes);
```

Comment:
Unfortunately, the JSON formatting is super fragile.  Here we need to use the logic that's currently determining whether to print a comma after the "resident" stat, and replace that logic with a simple comma for "resident".